Drivers/Inc/CANConfig.h

```diff
@@ -15,7 +15,7 @@
  * Filter Lists for CarCAN and MotorCAN
 */
 #define NUM_CARCAN_FILTERS 6
```

Comment:
If this is the number of allowed IDs, is this 4 now?

Reply:
Yes, good catch. I will change this.

Reply:
This will also be addressed in #387.